api/tests/unit/domain/services/scoring/scoring-service_test.js

```diff
@@ -0,0 +1,84 @@
+const scoringService = require('../../../../../lib/domain/services/scoring/scoring-service');
+const {
+  PIX_COUNT_BY_LEVEL,
+  MAX_REACHABLE_LEVEL,
+  MAX_REACHABLE_PIX_BY_COMPETENCE,
+} = require('../../../../../lib/domain/constants');
+
+const { expect, domainBuilder } = require('../../../../test-helper');
+
+describe('Unit | Service | Scoring Service', () => {
+
+  describe('#calculateScoringInformationForCompetence', function() {
+
+    it('should return the information about pix score and level for one competence', () => {
+      // given
+      const knowledgeElements = [
+        domainBuilder.buildKnowledgeElement({ earnedPix: 3.7 }),
+        domainBuilder.buildKnowledgeElement({ earnedPix: 4.4 }),
+        domainBuilder.buildKnowledgeElement({ earnedPix: 1.2 }),
+      ];
+
+      const expectedScoring = {
+        realTotalPixScoreForCompetence: 9.3,
+        pixScoreForCompetence: 9,
+        currentLevel: 1,
+        pixAheadForNextLevel: 1,
+      };
+
+      // when
+      const scoring = scoringService.calculateScoringInformationForCompetence(knowledgeElements);
+
+      // then
+      expect(scoring).to.be.deep.equal(expectedScoring);
```

Comment:
je ne crois pas que le `be` soit nécessaire.
```suggestion
      expect(scoring).to.deep.equal(expectedScoring);
```